LocalSettings.php

```diff
@@ -186,12 +186,36 @@
 	'wgCosmosWikiHeaderWordmark' => [
 		'default' => null,
 	],
-	'wgCosmosWikiHeaderBackgroundImage' => [
+	'wgCosmosWikiHeaderWordmark' => [
+		'default' => null,
+	],
+	'wgCosmosBackgroundImage' => [
 		'default' => null,
 	],
+	'wgCosmosBackgroundImageSize' => [
+		'default' => 'cover',
+	],
 	'wgCosmosMainBackgroundColor' => [
 		'default' => '#1A1A1A',
 	],
+	'wgCosmosContentBackgroundColor' => [
+		'default' => '#000',
+	],
+	'wgCosmosBannerBackgroundColor' => [
+		'default' => '#001e3b',
+	],
+	'wgCosmosLinkColor' => [
+		'default' => '#006cb0',
+	],
+	'wgCosmosBackgroundImageNorepeat' => [
```

Comment:
overridedefault is true but here the default is false

Reply:
That's intential. Paladox told me that that's how the boolean values have to work, as he changed all the other ones to do the same.